cl/compile.go

```diff
@@ -483,6 +483,30 @@ func (p *pkgCtx) recoverErr(e any, src ast.Node) error {
 	return err
 }
 
+// lookupClassFile looks up the class file by name.
+func (p *pkgCtx) lookupClassNode(name string) ast.Node {
+	for f, cls := range p.classes {
+		if name == cls.clsfile {
+			return &EmptyNode{
```

Comment:
Now f.Name is a implicit package name and has correct `End()` position.
See https://github.com/goplus/gop/pull/2354